grid2op/Parameters.py

```diff
@@ -26,7 +26,7 @@ class Parameters:
         thermal
         limit. Default is ``False``
 
-    NB_TIMESTEP_POWERFLOW_ALLOWED: ``int``
+    DNB_TIMESTEP_OVERFLOW_ALLOWED: ``int``
```

Comment:
Drum n Bass ? Or typo ?

Reply:
hahah it was a typo and the doc is not "unit tested" so i didn't catch that. I fixed it.